### DISCUSSION TITLE
https://github.com/pulls Showing No Pull Requests

### DISCUSSION THREAD

**Discussion by adamddh** [community/community#122757] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

When I visit https://github.com/pulls, I see no pull requests, even though I have many open and closed PRs.

**Comment by amyAMeQ** [comment]
Same here. I rely on that page quite a bit.  I hope it's not going away.

**Comment by easygolabs** [comment]
Similar problem for the `assigned`/`created` issues. Also, it is a visible freezing when loading and nothing loaded in a result...

**Reply by easygolabs** [reply]
For me - fixed just now. And.... still not working after refresh

**Comment by adamddh** [comment, marked as answer]
https://status.github.com is finally showing performance issues.
